src/poliastro/plotting/core.py

```diff
@@ -123,6 +124,13 @@ def plot_ephem(self, ephem, epoch=None, *, label=None, color=None, trail=False):
             Fade the orbit trail, default to False.
 
         """
+
+        if self._frame is None:
```

Comment:
This applies only for `OrbitPlotter2D`, not for `OrbitPlotter3D`, therefore it should not be in `_PlotlyOrbitPlotter`